src/codec.ts

```diff
@@ -457,6 +483,10 @@ function encodeValue(value: Value): Value {
     return value.map(encodeValue);
   }
 
+  if (value instanceof PGJsonb) {
+    return JSON.stringify(value.value);
```

Comment:
Can we use toString() method instead from PGJsonb class?

Reply:
Done